src/coffee/mapping.coffee

```diff
@@ -334,7 +334,7 @@ class Mapping
     errorMsg = "[row #{rowIndex}:#{attribName}] The value '#{rawBoolean}' isn't a valid boolean!"
     try
       b = JSON.parse(rawBoolean.toLowerCase())
-      if _.isBoolean(b) or _.isNumber(b)
+      if _.isBoolean(b) or b == 0 or b == 1
```

Comment:
Are we expecting this to be a number or string because 

```js
Boolean(1) // - outputs true
Boolean('1') // - outputs true
Boolean(0) // - outputs false
Boolean('0') // - outputs true
```